tattoo.ts

```diff
@@ -420,10 +430,29 @@ async function _main(elements: ElementRepo[]) {
     "repos/mocha",
     "repos/marked"
   ]);
-  const testProgress =
-      standardProgressBar("Testing...", elements.length);
   const testPromises: Array<Promise<TestResult>> = [];
-  for (const element of elements) {
+
+  let elementsToTest: ElementRepo[];
+
+  if (typeof opts["test-repo"] === "string") {
+    if (opts["test-repo"]) {
+      opts["test-repo"] = [opts["test-repo"]];
+    } else {
+      opts["test-repo"] = [];
+    }
+  }
+  if (opts["test-repo"].length > 0) {
+    elementsToTest = elements.filter((el) => {
+      return opts["test-repo"].indexOf(el.dir.substring(6)) > -1;
```

Comment:
what's `el.dir.substring(6)`? can you make a constant to make that more clear: `el.dir.substring(somethingPrefix)`?